### ISSUE TITLE
afk kill

### ISSUE THREAD

**Issue by agentecool** [overcore-gg/T1T4N#12]
this bug is when a team capture all points and the enemy team is afk, when the round restart the players can kill afks players

**Comment by mysterioan** [comment]
Which server/gamemode are you referring to?

**Comment by agentecool** [comment]
i am talking about the orangex10 100% crits

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

2017-12-01 14:09 GMT-05:00 mysterioan <notifications@github.com>:

**Comment by mysterioan** [comment]
I don't quite get the issue here. Can you rephrase it or elaborate?

**Comment by mysterioan** [comment]
No response for a week. Closed.
